reconcile/gitlab_housekeeping.py

```diff
@@ -119,8 +119,12 @@ def handle_stale_items(dry_run, gl, days_interval, enable_closing, item_type):
     for item in items:
         item_iid = item.attributes.get("iid")
         item_labels = item.attributes.get("labels")
-        if AUTO_MERGE in item_labels and item.merge_status == MRStatus.CANNOT_BE_MERGED:
-            close_item(dry_run, gl, enable_closing, item_type, item)
+        if AUTO_MERGE in item_labels:
+            if item.merge_status == MRStatus.UNCHECKED:
+                # this call triggers a status recheck
+                item = gl.get_merge_request(item_iid)
+            if item.merge_status == MRStatus.CANNOT_BE_MERGED:
```

Comment:
nit: IMHO `elif` would make it a little easier to read

Reply:
we are getting a new item to process, so an `elif` would not be logically correct?

Reply:
Whoops. Never would have expected that and thus overlooked it.